+++
# Projects widget.
widget = "projects"
active = true
date = 2016-04-20T00:00:00

title = "Research"
subtitle = ""

# Order that this section will appear in.
weight = 2

# Content.
# Display content from the following folder.
# For example, `folder = "project"` displays content from `content/project/`.
folder = "project"

# View.
# Customize how projects are displayed.
# Legend: 0 = list, 1 = cards.
view = 1

# Filter toolbar.

# Default filter index (e.g. 0 corresponds to the first `[[filter]]` instance below).
filter_default = 0

# Add or remove as many filters (`[[filter]]` instances) as you like.
# Use "*" tag to show all projects or an existing tag prefixed with "." to filter by specific tag.
# To remove toolbar, delete/comment all instances of `[[filter]]` below.
# [[filter]]
#   name = "All"
#   tag = "*"
#  
# [[filter]]
#   name = "Deep Learning"
#   tag = ".deep-learning"
#
# [[filter]]
#   name = "Other"
#   tag = ".demo"

+++

<div style="text-align: center"> 
<img src="/img/microbiome_influences_diagram_2_orange.png" width="500" height="500" usemap="#image-map">

<map name="image-map">
    <area target="" alt="diet" title="diet" href="project/3_host_diet_environment/" coords="206,59,295,84" shape="rect">
    <area target="" alt="environment" title="environment" href="project/3_host_diet_environment/" coords="322,260,497,282" shape="rect">
    <area target="" alt="genotype" title="genotype" href="project/2_host_genetics/" coords="9,133,149,158" shape="rect">
</map>
</div>

My long-term research interests revolve around understanding the complex interactions between the microbiome and the human host. 
Both my academic and industry experiences have led me to studying these complex interactions from a genomic perspective. 
As an undergraduate, I developed a strong interest in bacteriology while studying at the [University of Wisconsin-Madison](www.wisc.edu) (Go Badgers!). 
After graduating, I gained experience in industry working for [Roche Nimblegen](https://sequencing.roche.com/en/products-solutions/by-category/target-enrichment/hybridization.html), a microarray company. 
This experience introduced me to genomic techniques and I became proficient in the experimental skills necessary to perform high-throughput genomic research.

As a graduate student, I combined both my interests in microbial ecology and genomics by studying the human microbiome with [Yoav Gilad](www.giladlab.uchicago.edu) at the [University of Chicago](www.uchicago.edu). 
Specifically, I’ve led projects examining the role of [environmental](project/3_host_diet_environment/) and [genetic](project/2_host_genetics/) factors on determining gut microbiome composition.

As a postdoc with [Andy Clark](http://blogs.cornell.edu/clarklabblog/), I’ve continued to examine the role the [host genome](project/2_host_genetics/) plays in determining gut microbiome composition. 
In addition, I’m leading projects examining [microbe-microbe interactions in health and disease](project/1_current/#networks) and [characterizing the microbiomes of non-US/European populations](project/1_current/#nepal).

I’m now a jane-of-all-trades: doing all of my own experimental work and computational analyses. 
In addition, I’m interested in whether we’re studying the microbiome in the best way possible, and I’ve contributed to a [project](/publication/2013_taxonomic_classification_of_bacterial_16s/) aimed at methods development for microbiome studies and an [educational article](publication/2016_git/) for computational research methods.

**Read more here:**

