+++
title = "Current projects"
date = 2018-07-03T17:17:33+02:00
draft = false

# Tags: can be used for filtering projects.
# Example: `tags = ["machine-learning", "deep-learning"]`
tags = []

# Project summary to display on homepage.
summary = ""

# Optional image to display on homepage.
image_preview = "microbiome_current_fig.png"

# Optional external URL for project (replaces project detail page).
external_link = ""

# Does the project detail page use math formatting?
math = false

# Does the project detail page use source code highlighting?
highlight = true

# Featured image
# Place your image in the `static/img/` folder and reference its filename below, e.g. `image = "example.jpg"`.
[header]
image = ""
caption = ""

+++
Lots of fun microbiome science in the works!
Here are the projects I'm actively working on at the moment:

<img style="margin: 10px; float: left" src="/img/microbiome_current_fig.png" width="100" height="100">  [Co-occurrence network modeling reveals disease-specific configurations of microbiome community structure](#networks)

<br clear="all">

<img style="margin: 10px; float: left" src="/img/qqplot_BMI_genetics_and_genetics_microbiome.png" width="100">  [Simultaneously modeling host genetics and microbiome composition reveals the heritability and the proportion of variance explained due to the microbiome of immune-related traits](#microbiability)

<img style="margin: 10px; float: left" src="/img/nepal_pops.png" width="100">  [The microbiomes of Nepali populations practicing different subsistence strategies](#nepal)

<a name = "networks"></a>

<br>
<br>

## <span class="align_left"><i class="fa fa-angle-double-left fa-1x"></i><b>Co-occurrence network modeling reveals disease-specific configurations of microbiome community structure</b><i class="fa fa-angle-double-right fa-1x"></i></span>

<img style="margin: 10px; float: left" src="/img/microbiome_current_fig.png" width="300" height="300">  

The human gut microbiome is associated with a large number of complex traits and diseases. 
Although certain bacterial taxa have been identified that correlate with disease, it is commonly thought that alterations in _microbial community dynamics_ are indicative of a disease state. 
To investigate this, I'm examining how microbial community structure shifts between healthy and disease states for ~180 immune-related diseases and quantitative traits in a cohort of 2,500 twins from the United Kingdom using systems biology techniques. 
Using co-occurrence networks, I'm investigating both whether network-level properties differ between i) taxa associated with disease (centrality, connectedness, etc.) and ii) whether overall topology differs between networks built with affected or unaffected individuals (modularity, transitivity, etc.). 
My goal is to identify consistent changes in the microbiome community structure in health versus disease, across either all or a subset of the phenotypes I'm examining.

Analysis is still on-going to identify subtypes of disease with similar network properties. 
Additional datasets are also being incorporated to replicate results.
I'm always looking for new directions to take this research, so if you have any ideas or want to chat more, contact me!
 
I've presented this work at the following conferences:

- [Biology of Genomes (BoG) - 2017](https://meetings.cshl.edu/meetings.aspx?meet=genome&year=17)
- Society of Molecular Biology and Genetics (SMBE) - 2017
- [American Society of Human Genetics (ASHG) - 2017](http://www.ashg.org/2017meeting/)

<a name = "microbiability"></a>

<br>
<br>

## <span class="align_left"><i class="fa fa-angle-double-left fa-1x"></i><b>Simultaneously modeling host genetics and microbiome composition reveals the heritability and the proportion of variance explained due to the microbiome of immune-related traits</b><i class="fa fa-angle-double-right fa-1x"></i></span>

<img style="margin: 10px; float: left" src="/img/qqplot_BMI_genetics_and_genetics_microbiome.png" width="500" height="300">  

Both host genetics and environmental factors determine human immune-related phenotypes and disease, including asthma, allergies, and rheumatic disorders. 
Although many known environmental exposures contribute to immune disease risk and severity, the extent to which the human microbiome is responsible for variation in these phenotypes remains largely unknown. 
Additionally, whether gene-by-environment interactions with the microbiome influence these phenotypes remains uncharacterized, as well as which specific genetic variants and microbes play roles in this process. 

To address these gaps, I am examining both the proportion of variation explained by host genetics (PVE-G, or heritability) and gut microbiome composition (PVE-M) in a unified framework for ~30 immune-related phenotypes using the TwinsUK cohort.
Additionally, I'm exploring whether explicitly including gut microbiome composition in genome-wide association study (GWAS) models improves power to detect genetic variants associated with immune phenotypes. 

I am presenting this work at the following conferences:

- [American Society of Human Genetics (ASHG) - 2018](http://www.ashg.org/2018meeting/) (abstract submitted)
- [Probabilistic Modeling in Genomics (ProbGen) - 2018](http://meetings.cshl.edu/Probabilistic18) (session co-chair)

<a name = "nepal"></a>

<br>
<br>

## <span class="align_left"><i class="fa fa-angle-double-left fa-1x"></i><b>The microbiomes of Nepali populations practicing different subsistence strategies</b><i class="fa fa-angle-double-right fa-1x"></i></span>

<img style="margin: 10px; float: left" src="/img/nepal_fig.png" width="300" height="300">  
I'm collaborating with [Dr. Aashish Jha](https://profiles.stanford.edu/aashish-jha) to examine the microbiome in a fascinating set of populations from Nepal that in the past practiced semi-nomadic hunting and gathering. 
Some of these populations maintain that lifestyle (the Chepang), some took agricultural practices hundreds of years ago (the Tharu), and some have more recently transitioned (the Raute and Raji). 
Our goal is to identify whether these pre-industrial changes in subsistence strategies potentially influenced either the gut or oral microbiomes. 
While analysis of the oral microbiomes is on-going, gut microbiomes clearly differentiate between the subsistence groups. 
Incorporation of survey data reveal several lifestyle and dietary factors that may contribute, including drinking water source. 
More analysis is in the pipe for this study, so stay tuned!

Check out the preprint for our first study of Nepali gut microbiomes:

- [Gut microbiome transition across a lifestyle gradient in Himalaya](https://www.biorxiv.org/content/early/2018/01/27/253450)

Also check out Aashish's website describing the project:

- [Himalayan Diversity Project](http://www.himalayandiversityproject.org/about.html)

